lib/foreman_monitoring/engine.rb

```diff
@@ -86,14 +86,12 @@ class Engine < ::Rails::Engine
     end
 
     config.to_prepare do
-      begin
         ::Host::Managed.prepend(ForemanMonitoring::HostExtensions)
         ::Hostgroup.include(ForemanMonitoring::HostgroupExtensions)
         ::HostsHelper.prepend(ForemanMonitoring::HostsHelperExt)
         ::HostsController.prepend(ForemanMonitoring::HostsControllerExtensions)
       rescue StandardError => e
         Rails.logger.warn "ForemanMonitoring: skipping engine hook (#{e})"
```

Comment:
Please fix the indenting here if you're removing the begin/end parts.

Reply:
@ekohl it is updated now.